Makefile

```diff
@@ -173,6 +176,11 @@ $(LINK_CHECKER): $(TOOLS_DIR)/go.mod
 $(GO_APIDIFF): $(TOOLS_DIR)/go.mod
 	cd $(TOOLS_DIR) && go build -tags=tools -o $(GO_APIDIFF_BIN) github.com/joelanford/go-apidiff
 
+$(ENVSUBST): $(TOOLS_DIR)/go.mod
+	cd $(TOOLS_DIR) && go build -tags=tools -o $(ENVSUBST_BIN) github.com/drone/envsubst/cmd/envsubst
```

Comment:
we should do the same in capz, was wondering about the impact the subtle differences between the two could have as well.